File: codewars/python/8kyu/chromosome_check.py
def chromosome_check(sperm: str):
    if 'Y' in sperm:
        return "Congratulations! You're going to have a son."
    return "Congratulations! You're going to have a daughter."

if __name__ == '__main__':
    print(chromosome_check('XY'))
    print(chromosome_check('XX'))

File: codewars/dart/8kyu/chromosome_check.dart
void main(List<String> args) {
  print(chromosome_check('XY'));
  print(chromosome_check('XX'));
}

String chromosome_check(String sperm) {
  if (sperm.contains('Y')) {
    return "Congratulations! You're going to have a son.";
  }
  return "Congratulations! You're going to have a daughter.";
}
